source/_integrations/mystrom.markdown

```diff
@@ -167,23 +148,5 @@ Make sure that you have enabled the REST API under **Advanced** in the web front
   <img src='/images/integrations/mystrom/switch-advanced.png' />
 </p>
 
-To use your myStrom switch in your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-switch:
-  - platform: mystrom
-    host: IP_ADRRESS
-```
+{%include integrations/config_flow.md %}
```

Comment:
This setup instructions should only appear once for this integration?

Reply:
merged with the light description